src/godot_sqlite.cpp

```diff
@@ -288,30 +288,26 @@ SQLite::~SQLite() {
 
 void SQLite::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("open", "database"), &SQLite::open);
-	ClassDB::bind_method(D_METHOD("open_from_path", "path"), &SQLite::open_from_path);
 	ClassDB::bind_method(D_METHOD("open_in_memory"), &SQLite::open_in_memory);
-	ClassDB::bind_method(D_METHOD("open_buffered", "path", "buffers", "size"),
-			&SQLite::open_buffered);
+	ClassDB::bind_method(D_METHOD("open_buffered", "path", "buffers", "size"), &SQLite::open_buffered);
+	ClassDB::bind_method(D_METHOD("backup", "path"), &SQLite::backup);
```

Comment:
Is backup the same as save?

Reply:
oh, it's without the resource parameter.

`save(const Ref<Resource> &p_resource, const String &p_path, uint32_t p_flags)`